File: C#/Basic logic/B/B4.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace B
{
    internal class B4
    {
        static void Mainx(string[] args)
        {
            Console.WriteLine("zadej 3 čísla");
            int cislo1 = int.Parse(Console.ReadLine());
            int cislo2 = int.Parse(Console.ReadLine());
            int cislo3 = int.Parse(Console.ReadLine());

            if (cislo1 > cislo2 && cislo1> cislo3)
                Console.WriteLine("číslo " + cislo1 + " je největší");
            if (cislo2> cislo1 && cislo2>cislo3)
                Console.WriteLine("číslo " + cislo2 + " je největší");
            if (cislo3 > cislo1 && cislo3 > cislo2)
                Console.WriteLine("číslo " + cislo3 + " je největší");
            if (cislo1 < cislo2 && cislo1 < cislo3)
                Console.WriteLine("číslo " + cislo1 + " je nejmenší");
            if (cislo2 < cislo1 && cislo2 < cislo3)
                Console.WriteLine("číslo " + cislo2 + " je nejmenší");
            if (cislo3 < cislo1 && cislo3 < cislo2)
                Console.WriteLine("číslo " + cislo3 + " je nejmenší");


        }
    }
}

File: Java/Basics To Java/src/B/B4.java
package B;

import java.util.Scanner;

public class B4 {
	public static void mainx(String[] args) {
		Scanner sc = new Scanner(System.in);

		System.out.println("zadje 3 èísla");
		int cislo1= sc.nextInt();
		int cislo2= sc.nextInt();
		int cislo3= sc.nextInt();
		
        if (cislo1 > cislo2 && cislo1> cislo3)
            System.out.println("èíslo " + cislo1 + " je nejvìtší");
        if (cislo2> cislo1 && cislo2>cislo3)
        	System.out.println("èíslo " + cislo2 + " je nejvìtší");
        if (cislo3 > cislo1 && cislo3 > cislo2)
        	System.out.println("èíslo " + cislo3 + " je nejvìtší");
        if (cislo1 < cislo2 && cislo1 < cislo3)
        	System.out.println("èíslo " + cislo1 + " je nejmenší");
        if (cislo2 < cislo1 && cislo2 < cislo3)
        	System.out.println("èíslo " + cislo2 + " je nejmenší");
        if (cislo3 < cislo1 && cislo3 < cislo2)
        	System.out.println("èíslo " + cislo3 + " je nejmenší");
	}
}
